samples/memfs/memfs.go

```diff
@@ -194,6 +194,10 @@ func (fs *memFS) StatFS(
 func (fs *memFS) LookUpInode(
 	ctx context.Context,
 	op *fuseops.LookUpInodeOp) error {
+	if op.OpContext.Pid == 0 {
+		return fuse.EINVAL
+	}
+
```

Comment:
@kahing Do you think this is a good way to test whether pid is there? (would like to pick your brain before adding this to all operations).

Tests in `memfs_test.go` all call `os` methods, without calling methods here directly. Ideally I was thinking about comparing pid from `os.GetPid()` to the pid in `LookUpInodeOp`, but didn't quite see how we can achieve that. Appreciate some thoughts!

Reply:
I think this is fine. 

One way to test the actual pid is by first pinning test goroutine to one OS thread (via https://golang.org/pkg/runtime/#LockOSThread). Then we can collect all the pids `MemFS` ever sees, and then check that at the end of the tests.